src/makeInfinityAdder.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
```

Comment:
```suggestion
  return function (number) {
```
if you return function right away you can skip name